docs/resources/ddm_schema.md

```diff
@@ -84,7 +86,7 @@ resource "huaweicloud_ddm_schema" "test"{
   data_nodes {
     id             = huaweicloud_rds_instance.test.id
     admin_user     = "root"
-    admin_password = "test_password_123"
+    admin_password = var.admin_password
```

Comment:
1. `admin_password` is not defined, it should be `var.password`
2. `password` is not clear， suggest to rename as `rds_password`

Reply:
It has been fixed.